phys2denoise/metrics/chest_belt.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from scipy.interpolate import interp1d
+
+
+def rvt(belt_ts, peaks, troughs, samplerate, lags=(0, 4, 8, 12)):
+    """ "
+    Implements the Respiratory Variance over Time as described in Birn et al. (2006).
+    Procedural choices influenced by RetroTS
+
+    Parameters
+    ----------
+    belt_ts: array_like
+        respiratory belt data - samples x 1
+    peaks: array_like
+        peaks found by peakdet algorithm
+    troughs: array_like
+        troughs found by peakdet algorithm
+    samplerate: float
+        sample rate in hz of respiratory belt
+    lags: tuple
+        lags in seconds of the RVT output. Default is 0, 4, 8, 12.
+
+    Outputs
+    -------
+    rvt: array_like
+        calculated RVT and associated lags.
+
+    """
+
+    timestep = 1 / samplerate
+    # respiration belt timing
+    time = np.array([i * timestep for i in range(len(belt_ts))])
```

Comment:
I think using `np.arange` would be cleaner.

Reply:
:+1:
`np.arange` or even `np.linspace` using the samplerate (right @tsalo?)
 
(Bonus: it should be much faster!)

Reply:
Done!